services/web/client/source/class/qxapp/component/workbench/NodeUI.js

```diff
@@ -204,32 +211,13 @@ qx.Class.define("qxapp.component.workbench.NodeUI", {
     },
 
     getLinkPoint: function(port) {
-      let nodeBounds = this.getCurrentBounds();
-      if (nodeBounds === null) {
-        // not rendered yet
-        return null;
+      const bounds = this.getCurrentBounds();
+      const captionHeight = 20; // HARDCODED: Otherwise the caption height is changing and misplacing the links
```

Comment:
I guess you can get that number from the Appearance of the window ("window-small-cap/captionbar")

Reply:
Genius!! I hadn't seen that there was a minHeight there. So that is why it's changing. Its natural height is 15 and then it applies the minHeight. Makes sense right? Other option is to simply remove that from the theme...

Reply:
if there is no text in the caption the caption will become very small if there is no minHeight, but depending on the design this might be no problem